docker/Dockerfile

```diff
@@ -2,30 +2,18 @@ ARG UBUNTU_VERSION=18.04
 FROM ubuntu:${UBUNTU_VERSION}
 
 # Use bash as the default shell
-RUN ln -snf /bin/bash /bin/sh
+SHELL ["/bin/bash", "-c"]
 
 # Update core packages
 RUN apt-get -y update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
 
 # Get some apt packages
 RUN apt-get -y install autoconf clang cmake g++ gcc gfortran git libblas-dev \
                        libhdf5-dev liblapack-dev libpython2.7-dev libtool \
                        python-numpy python-pip python-setuptools wget
 
-# Get astyle
-RUN if [ "${UBUNTU_VERSION}" == "18.04" ]; then \
-      apt-get -y install astyle; \
-    else \
-      astyle_deb=astyle_3.1-1ubuntu2_amd64.deb && \
-      wget https://launchpad.net/ubuntu/+source/astyle/3.1-1ubuntu2/+build/14532685/+files/${astyle_deb} && \
-      dpkg -i ${astyle_deb} && \
-      rm -f ${astyle_deb}; \
-    fi
-
 # Get some pip packages
-RUN pip install cython sphinx
+RUN pip install cython
```

Comment:
Why was sphinx moved out of this file?

Reply:
It looks like anything having to do w/ housekeeping was moved to the housekeeping.sh file.